apps/workflows/jobs/refill-daily.ts

```diff
@@ -29,14 +33,32 @@ client.defineJob({
             gt(table.refillAmount, table.remaining),
             or(
               isNull(table.lastRefillAt),
-              lte(table.lastRefillAt, t), // Check if more than 24 hours have passed
+              lte(table.lastRefillAt, tDay), // Check if more than 24 hours have passed
             ),
           ),
       }),
     );
-    io.logger.info(`found ${keys.length} keys with daily refill set`);
+    const monthlyKeys = await io.runTask("list keys Monthly", () =>
```

Comment:
we talked about treating all keys the same, no?
why do we run 2 queries here one for monthly and one for daily

Reply:
combined